mods/gen2/formats-data.js

```diff
@@ -119,21 +115,22 @@ exports.BattleFormatsData = {
 		},
 		tier: "NFE",
 	},
-	charizard: {
-		randomSet1: {
+	charizard: {//done
```

Comment:
I think we mostly agreed on zard so I leave it like that. Only thing is I don't think the fighting move is worth it.